react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,212 @@
+/* @flow */
+
+import React from 'react';
+
+declare var APP: Object;
+declare var JitsiMeetJS: Object;
+
+/**
+ * All toolbar buttons' descriptions.
+ */
+export default {
+    'microphone': {
+        id: 'toolbar_button_mute',
```

Comment:
I left `id` property in these objects because we use id to set button tooltips. But actual differentiation between buttons is now implemented via their names in `interface_config`